src/routes/index.svelte

```diff
@@ -2,6 +2,12 @@
   import { Field } from '$lib/db';
   import Leaflet from '$lib/leaflet/Leaflet.svelte';
   import GeoJson from '$lib/leaflet/GeoJson.svelte';
+  import { goto } from '$app/navigation';
+
+
+const handle_click_event = (field : Field) => {
```

Comment:
Can you make the name more meaningful, maybe `goToField` ?

Please use an actual function rather then anonymous one.